src/main/resources/application.yml

```diff
@@ -168,3 +168,8 @@ kafka:
       topic-pattern: (${environment}\.)(.*\.)inventory\.holdings-record
       concurrency: ${KAFKA_EVENTS_CONCURRENCY:2}
       data-type: org.folio.innreach.dto.Holding
+  backoff:
+    interval: ${DEFAULT_INTERVAL:95000}
+    max_failure: ${MAX_FAILURE:360}
```

Comment:
Noticing that there is a section in the application.yml called `batch.jobs.contribution` which has items like `retry-attempts` and `retry-interval-ms`. These properties are used in the `ContributionJobProperties` as `retryAttempts` and `retryIntervalMs`. Are these being used?

If not, it seems like we need to do one of two things:
1. Remove the `batch.jobs.contribution` section and related code completely if it isn't being used.
2. Remove the `kafka.backoff` section of the yaml and instead use the `batch.jobs.contribution` properties instead. Seems like the word "batch" is used elsewhere as a namespace for "initial contribution" job stuff. So this might be better.